File: 프로그래머스/unrated/181888. n개 간격의 원소들/n개 간격의 원소들.cpp
#include<bits/stdc++.h>
using namespace std;
vector<int> ret;
vector<int> solution(vector<int> num_list, int n) {
    
    for(int i = 0; i < num_list.size(); i++){
        
        if(i % n == 0) ret.push_back(num_list[i]);
    }
    
    return ret;
}

File: 프로그래머스/0/181888. n개 간격의 원소들/n개 간격의 원소들.java
import java.util.*;
class Solution  {
    public List<Integer> solution(int[] num_list, int n) {
        List<Integer> ret = new ArrayList<>();
        
        for(int i = 0; i < num_list.length; i += n){
            ret.add(num_list[i]);
        }
        
        return ret;
    }
}